File: src/neetcode/Java/_1TwoSum.java
package neetcode.Java;

public class _1TwoSum {
    public static int[] twoSum(int[] nums, int target) {
        for (int i = 1; i < nums.length; i++){
            for (int j = 0; j + i < nums.length; j++) {
                if (nums[j+i] + nums[j] == target) {
                    return new int[]{j, j+i};
                }
            }
        }
        return new int[]{};
    }

    public static void main(String[] args) {
        int[] num={2,7,11,15};
        int a[] = twoSum(num,9);
        System.out.println(a[0]+" "+a[1]);
    }
}

File: src/neetcode/kotlin/_1TwoSum.kt
package neetcode.kotlin

object _1TwoSum {
    fun twoSum(nums: IntArray, target: Int): IntArray {
        for (i in 1 until nums.size) {
            var j = 0
            while (j + i < nums.size) {
                if (nums[j + i] + nums[j] == target) {
                    return intArrayOf(j, j + i)
                }
                j++
            }
        }
        return intArrayOf()
    }

    @JvmStatic
    fun main(args: Array<String>) {
        val num = intArrayOf(2, 7, 11, 15)
        val a = twoSum(num, 9)
        println(a[0].toString() + " " + a[1])
    }
}
